### PR TITLE
TUI: Add j,k vim bindings for infopane control

### PR DESCRIPTION
**Pull request by ndzik**
Adds `j` to move "downwards" and `k` to move "upwards".

I figured I cannot be the only terminal enthusiast who loves to never leave the homerow. I realize this is opinionated but I could not live without it (':

PS: I did not use any formatter, so I aligned stuff the way ~~my~~ I thought would be okay for now.

BTW: I just love everything about this project :rocket:

### CODE REVIEW

File: src/Swarm/TUI/View.hs [hunk 1]
```diff
@@ -239,7 +239,7 @@ drawMenu isPaused viewingBase mode
       [ ("c", "recenter") | not viewingBase ]
 
     keyCmdsFor (Just InfoPanel)  =
-      [ ("↓↑/Pg{Up,Dn}/Home/End", "navigate")
+      [ ("↓↑/Pg{Up,Dn}/Home/End/{j,k}", "navigate")
```
Comment (TristanCacqueray) [inline src/Swarm/TUI/View.hs:242]: Should we use the same help as the world panel, e.g. `↓↑ / jk` ?

**Review by TristanCacqueray — APPROVED** [review]
Thanks!

File: src/Swarm/TUI/View.hs (hunk 1 above)
Comment (ndzik) [reply src/Swarm/TUI/View.hs:242]: I also could not decide. I have no problems with a version using less characters. Will update it!

**Review by byorgey — APPROVED** [review]

**Comment by byorgey** [comment]
> BTW: I just love everything about this project :rocket:

Thanks, glad you like it! =)

**Comment by byorgey** [comment]
@ndzik  , congrats on having your first PR accepted to Swarm!  We're really glad to welcome you as part of the [community](https://github.com/byorgey/swarm/blob/main/COMMUNITY.md).  In a minute we will send you an invite to the repo (see https://github.com/byorgey/swarm/blob/main/CONTRIBUTING.md#i-have-push-access-to-the-swarm-repository-now-what ).  If you use IRC, feel free to also join the `#swarm` channel on Libera.Chat.

**Pull request merged**